server/models/Feed.js

```diff
@@ -58,6 +60,8 @@ class Feed extends Model {
     this.explicit
     /** @type {boolean} */
     this.preventIndexing
+    /** @type {boolean} */
+    this.reverseOrder
```

Comment:
If the database is changing, we will need to have an accompanying database migration script to support rolling back to previous database versions.

More information can be found here https://github.com/advplyr/audiobookshelf/blob/master/server%2Fmigrations%2Freadme.md

I'm not sure if this table has anything referencing it or if we can just use `changeColumn`. We have had some issues with `changeColumn` in the past.

Reply:
`v2.17.6-share-add-isdownloadable` looks like a perfect template. Added!